keps/sig-apps/2255-pod-cost/README.md

```diff
@@ -55,191 +49,216 @@ Items marked with (R) are required *prior to targeting to a milestone / release*
 
-When upgrading no changes are needed to maintain existing behaviour as all of this behaviour is fully optional and disabled by default. To activate this feature either a user has to make an annotation to a pod in a Deployment by hand or the application annotates a pod in a Deployment through the API.
-
-When downgrading, there is no need to changing anything, as this is just a pod annotation, which is uncritical.
+There is no strategy per se. On upgrade, controller-manager will start taking into account
+pod-deletion-cost annotation for new and existing ReplicaSets that set the annotation. On 
+downgrade, controller-manager will stop taking into account pod-deletion-cost, and so
+reverting to old behavior.
 
 ### Version Skew Strategy
 
-As this feature is based on pod annotations, there is no issue with different Kubernetes versions. The lack of this feature in older versions may change the efficiency and reliability of the applications.
+N/A
 
 ## Production Readiness Review Questionnaire
 
 ### Feature enablement and rollback
 
 * **How can this feature be enabled / disabled in a live cluster?**
-  - [x] Other
-    - Make special pod annotations within a live Deployment
+  - [x] Feature gate (also fill in values in `kep.yaml`)
+    - Feature gate name: ReplicaSetPodDeletionCost
+    - Components depending on the feature gate: kube-controller-manager
+  - [ ] Other
+    - Describe the mechanism:
+    - Will enabling / disabling the feature require downtime of the control
+      plane?
+    - Will enabling / disabling the feature require downtime or reprovisioning
+      of a node? (Do not assume `Dynamic Kubelet Config` feature is enabled).
 
 
 * **Does enabling the feature change any default behavior?**
-  - No
+ No.
 
 
 * **Can the feature be disabled once it has been enabled (i.e. can we rollback
   the enablement)?**
-  - One can either remove the annotations or downgrade to an older Kubernetes release
+Yes.
 
 
 * **What happens if we reenable the feature if it was previously rolled back?**
-  - Then the feature will be reenabled. Nothing special to consider here.
+It should continue to work as expected.
 
 
 * **Are there any tests for feature enablement/disablement?**
+No, but we can do manual testing.
+
+### Scalability
+
+* **Will enabling / using this feature result in any new API calls?**
+  - No, not the feature itself. However, users will want to deploy an external controller 
+    that updates the pod-deletion-cost, documentation should stress that update frequency
+    to be coarse grained.
+
+
+* **Will enabling / using this feature result in introducing new API types?**
+  - No.
+
+
+* **Will enabling / using this feature result in any new calls to cloud
+  provider?**
+  - No.
+
+
+* **Will enabling / using this feature result in increasing size or count
+  of the existing API objects?**
+  Describe them providing:
+  - API type(s): Pod annotation
+  - Estimated increase in size: Size of a new annotation
```

Comment:
You can point out that values are expected to be integers

Reply:
the key and value are strings, it is not a four byte int.